README.md

```diff
@@ -402,7 +402,7 @@ Output:
         thandoodstrenterprourete: [ '_reader', '_writer' ],
         nodejs: [ '_reader', '_writer', '_admin', '_replicator' ] } }
 
-See the [Authorization Documentation](https://console.bluemix.net/docs/services/Cloudant/api/authorization.html#authorization) for further details.
+See the [Authorization] for further details.
```

Comment:
`See the [Authorization] for further details.` 
`See [Authorization] for further details.`
or `See the [Authorization] documentation for further details.`